osu.Game.Rulesets.Tau/Judgements/TauJudgement.cs

```diff
@@ -6,8 +6,8 @@ namespace osu.Game.Rulesets.Tau.Judgements
     public class TauJudgement : Judgement
     {
         public override HitResult MaxResult => HitResult.Great;
-
-        protected override int NumericResultFor(HitResult result)
+
+        protected new int ToNumericResult(HitResult result)
```

Comment:
Don't use `new`, it hides the previous method, and ultimately serves no purpose in this case.

All rulesets follow by the standardized score values set by osu, so this can be removed